ansible/inventory-bootstrap.yaml

```diff
@@ -13,6 +13,7 @@ all:
       vars:
         ceramic_network: testnet-clay
         gcp_project: tnet-prod-2024
+        ssh_key_cd_pub: "{{ lookup('gcp_secret', gcp_project, 'bootstrap-tnet-prod-id_rsa-pub') }}"
```

Comment:
hm how do these keys get put in place?  i am using a local secrets file for my github token for the generic ansible playbook, should we be using a global one?